# 7월 19일 수요일 asnyc-await 문제입니다.

```
async function f() {
  let result = 'first!';
  let promise = new Promise((resolve, reject) => {
    setTimeout(() => resolve('done!'), 1000);
  });

  result = await promise;

  console.log(result);
}

f();
```

A: first!

B: done!

C: JavaScript error

D: Something else